### ISSUE TITLE
N application method

### ISSUE THREAD

**Issue by Galio23** [leombastos/2022_AdvPA#5]
Which method are we using to apply N in the field? Base fertilizer or through the irrigation system?

**Comment by emilykbedwell** [comment]
I have a question as well to complement Giannis': will the grower be using a split application method as we worked on in class?

**Comment by leombastos** [comment]
We are going to be using split application (base rate + at least one side-dress), with the variable rate approach being used only for the side-dress application (similarly to in class).

**Comment by emilykbedwell** [comment]
Yeah, .33 or 1/3 of the N will be set aside for the side-dress, if i understand correctly.

**Comment by Galio23** [comment]
I just saw the question asks for in-season only and not whole amount that we need to buy. Thanks for your time.

**Comment by leombastos** [comment]
In the example we did in class, we used 1/3 base rate (we called pre-plant in class), and 2/3 side-dress. This split ratio was the recommendation from the cotton guide (example in class).
